2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,57 @@
+#__BY:sirjan13__
+
+
+def gcd(n, m):
+    if n == 0:
+        return m
+    return gcd(m % n, n)
+
+
+t1, t2 = 0, 0
+
+
+def egcd(a, b):
+    x, y, u, v = 0, 1, 1, 0
+    while a != 0:
+        q, r = b//a, b % a
```

Comment:
Why such partiality with integer division binary operatory 😟

Reply:
actually i used a source for efficient gcd as mentioned in my codechef code they might have had it shall i remove this?